### PR TITLE
Use the clinicalgenomics/python3.11-venv:1.0 image everywhere in the Dockerfiles

### LINKED ISSUE #5083: Container doesn't start on stage
It has the following error: 

Build fails also locally, with this error: scout-cli          | ImportError: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /venv/lib/python3.11/site-packages/cryptography/hazmat/bindings/_rust.abi3.so)

Thank goodness prod is fine! 🙏🏻

### PR DESCRIPTION
**Pull request by northwestwitch**
This PR adds a functionality or fixes a bug.
- Fix #5083 

<details>
<summary>Testing on cg-vm1 server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. Make sure the PR is pushed and available on [Docker Hub](https://hub.docker.com/repository/docker/clinicalgenomics/scout-server-stage)
1. Fist book your testing time using the Pax software available at [https://pax.scilifelab.se/](https://pax.scilifelab.se). The resource you are going to call dibs on is `scout-stage` and the server is `cg-vm1`.
1. `ssh <USER.NAME>@cg-vm1.scilifelab.se`
1. `sudo -iu hiseq.clinical`
1. `ssh localhost`
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `podman ps`
1. Stop the service with current deployed branch: `systemctl --user stop scout.target`
1. Start the scout service with the branch to test: `systemctl --user start scout@<this_branch>`
1. Make sure the branch is deployed: `systemctl --user status scout.target`
1. After testing is done, repeat procedure at [https://pax.scilifelab.se/](https://pax.scilifelab.se), which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

<details>
<summary>Testing on hasta server (Clinical Genomics Stockholm)</summary>

**Prepare for testing**
1. `ssh <USER.NAME>@hasta.scilifelab.se`
1. Book your testing time using the Pax software. `us; paxa -u <user> -s hasta -r scout-stage`. You can also use the WSGI Pax app available at [https://pax.scilifelab.se/](https://pax.scilifelab.se).
1. (optional) Find out which scout branch is currently deployed on cg-vm1: `conda activate S_scout; pip freeze | grep scout-browser`
1. Deploy the branch to test: `bash /home/proj/production/servers/resources/hasta.scilifelab.se/update-tool-stage.sh -e S_scout -t scout -b <this_branch>`
1. Make sure the branch is deployed: `us; scout --version`
1. After testing is done, repeat the `paxa` procedure, which will release the allocated resource (`scout-stage`) to be used for testing by other users.
</details>

**How to test**:
1. how to test it, possibly with real cases/data

**Expected outcome**:
The functionality should be working
Take a screenshot and attach or copy/paste the output.

**Review:**
- [x] code approved by DNIL
- [x] tests executed by CR

### CODE REVIEW

**Comment by northwestwitch** [comment]
Nope, this one fails as well on stage. I'll investigare better locally first..

**Comment by dnil** [comment]
Yes, I see good start locally with "python3.12-venv" and "python:3.12-slim-bookworm"!

**Comment by dnil** [comment]
Ah, I think I know: `Dockerfile-server

**Comment by northwestwitch** [comment]
> Ah, I think I know: `Dockerfile-server

?

**Comment by dnil** [comment]
> > Ah, I think I know: `Dockerfile-server
> 
> ?

Ah, you did update it as well: sorry, I'm too distracted! But weird then?! Just the build platform difference.

**Comment by dnil** [comment]
```
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:   File "/venv/lib/python3.12/site-packages/gunicorn/util.py", line 370, in import_app
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:     mod = importlib.import_module(module)
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:     super().init_process()
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:   File "/venv/lib/python3.12/site-packages/gunicorn/workers/base.py", line 135, in init_process
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:     self.load_wsgi()
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:   File "/venv/lib/python3.12/site-packages/gunicorn/workers/base.py", line 135, in init_process
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:     self.load_wsgi()
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:   File "/venv/lib/python3.12/site-packages/gunicorn/workers/base.py", line 147, in load_wsgi
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:     self.wsgi = self.app.wsgi()
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:                 ^^^^^^^^^^^^^^^
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:   File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:   File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:   File "<frozen importlib._bootstrap_external>", line 995, in exec_module
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:   File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
Nov 28 15:41:10 cg-services-stage.scilifelab.se podman[3101366]:   permit_unconventional_http_method: False
Nov 28 15:41:10 cg-services-stage.scilifelab.se podman[3101366]:   permit_unconventional_http_version: False
Nov 28 15:41:10 cg-services-stage.scilifelab.se podman[3101366]:   casefold_http_method: False
Nov 28 15:41:10 cg-services-stage.scilifelab.se podman[3101366]:   forwarder_headers: ['SCRIPT_NAME', 'PATH_INFO']
Nov 28 15:41:10 cg-services-stage.scilifelab.se podman[3101366]:   header_map: drop
Nov 28 15:41:10 cg-services-stage.scilifelab.se podman[3101366]: [2024-11-28 15:41:10 +0100] [8] [INFO] Starting gunicorn 23.0.0
Nov 28 15:41:10 cg-services-stage.scilifelab.se podman[3101366]: [2024-11-28 15:41:10 +0100] [8] [DEBUG] Arbiter booted
Nov 28 15:41:10 cg-services-stage.scilifelab.se podman[3101366]: [2024-11-28 15:41:10 +0100] [8] [INFO] Listening at: http://0.0.0.0:8000 (8)
Nov 28 15:41:10 cg-services-stage.scilifelab.se podman[3101366]:   ssl_version: 2
Nov 28 15:41:10 cg-services-stage.scilifelab.se podman[3101366]:   cert_reqs: 0
Nov 28 15:41:10 cg-services-stage.scilifelab.se podman[3101366]:   ca_certs: None
Nov 28 15:41:10 cg-services-stage.scilifelab.se podman[3101366]:   suppress_ragged_eofs: True
Nov 28 15:41:10 cg-services-stage.scilifelab.se podman[3101366]:   do_handshake_on_connect: False
Nov 28 15:41:10 cg-services-stage.scilifelab.se podman[3101366]:   ciphers: None
Nov 28 15:41:10 cg-services-stage.scilifelab.se podman[3101366]:   raw_paste_global_conf: []
Nov 28 15:41:10 cg-services-stage.scilifelab.se podman[3101366]:   permit_obsolete_folding: False
Nov 28 15:41:10 cg-services-stage.scilifelab.se podman[3101366]:   strip_header_spaces: False
Nov 28 15:41:10 cg-services-stage.scilifelab.se podman[3101366]:   permit_unconventional_http_method: False
Nov 28 15:41:10 cg-services-stage.scilifelab.se podman[3101366]:   permit_unconventional_http_version: False
Nov 28 15:41:10 cg-services-stage.scilifelab.se podman[3101366]:   casefold_http_method: False
Nov 28 15:41:10 cg-services-stage.scilifelab.se podman[3101366]:   forwarder_headers: ['SCRIPT_NAME', 'PATH_INFO']
Nov 28 15:41:10 cg-services-stage.scilifelab.se podman[3101366]:   header_map: drop
Nov 28 15:41:10 cg-services-stage.scilifelab.se podman[3101366]: [2024-11-28 15:41:10 +0100] [8] [INFO] Starting gunicorn 23.0.0
Nov 28 15:41:10 cg-services-stage.scilifelab.se podman[3101366]: [2024-11-28 15:41:10 +0100] [8] [DEBUG] Arbiter booted
Nov 28 15:41:10 cg-services-stage.scilifelab.se podman[3101366]: [2024-11-28 15:41:10 +0100] [8] [INFO] Listening at: http://0.0.0.0:8000 (8)
Nov 28 15:41:10 cg-services-stage.scilifelab.se podman[3101366]: [2024-11-28 15:41:10 +0100] [8] [INFO] Using worker: gthread
Nov 28 15:41:10 cg-services-stage.scilifelab.se podman[3101366]: [2024-11-28 15:41:10 +0100] [9] [INFO] Booting worker with pid: 9
Nov 28 15:41:10 cg-services-stage.scilifelab.se podman[3101366]: [2024-11-28 15:41:10 +0100] [10] [INFO] Booting worker with pid: 10
Nov 28 15:41:10 cg-services-stage.scilifelab.se podman[3101366]: [2024-11-28 15:41:10 +0100] [11] [INFO] Booting worker with pid: 11
Nov 28 15:41:10 cg-services-stage.scilifelab.se podman[3101366]: [2024-11-28 15:41:10 +0100] [12] [INFO] Booting worker with pid: 12
Nov 28 15:41:10 cg-services-stage.scilifelab.se podman[3101366]: [2024-11-28 15:41:10 +0100] [8] [DEBUG] 4 workers
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]: 2024-11-28 15:41:12 5b1ac6ed234a scout.server.extensions.chanjo_extension[10] ERROR No module named 'pkg_resources'
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]: 2024-11-28 15:41:12 5b1ac6ed234a scout.server.extensions.chanjo_extension[11] ERROR No module named 'pkg_resources'
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]: [2024-11-28 15:41:12 +0100] [10] [ERROR] Exception in worker process
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]: Traceback (most recent call last):
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:   File "/venv/lib/python3.12/site-packages/gunicorn/arbiter.py", line 608, in spawn_worker
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:     worker.init_process()
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:   File "/venv/lib/python3.12/site-packages/gunicorn/workers/gthread.py", line 94, in init_process
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:     super().init_process()
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:   File "/venv/lib/python3.12/site-packages/gunicorn/workers/base.py", line 135, in init_process
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:     self.load_wsgi()
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:   File "/venv/lib/python3.12/site-packages/gunicorn/workers/base.py", line 147, in load_wsgi
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:     self.wsgi = self.app.wsgi()
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:                 ^^^^^^^^^^^^^^^
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:   File "/venv/lib/python3.12/site-packages/gunicorn/app/base.py", line 66, in wsgi
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:     self.callable = self.load()
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:                     ^^^^^^^^^^^
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:   File "/venv/lib/python3.12/site-packages/gunicorn/app/wsgiapp.py", line 57, in load
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:     return self.load_wsgiapp()
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:            ^^^^^^^^^^^^^^^^^^^
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:   File "/venv/lib/python3.12/site-packages/gunicorn/app/wsgiapp.py", line 47, in load_wsgiapp
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:     return util.import_app(self.app_uri)
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:   File "/venv/lib/python3.12/site-packages/gunicorn/util.py", line 370, in import_app
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:     mod = importlib.import_module(module)
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:   File "/usr/local/lib/python3.12/importlib/__init__.py", line 90, in import_module
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:     return _bootstrap._gcd_import(name[level:], package, level)
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:   File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:   File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:   File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:   File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:   File "<frozen importlib._bootstrap_external>", line 995, in exec_module
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:   File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:   File "/home/worker/app/scout/server/auto.py", line 13, in <module>
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:     app = create_app(config_file=os.environ["SCOUT_CONFIG"])
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:   File "/home/worker/app/scout/server/app.py", line 68, in create_app
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:     configure_extensions(app)
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:   File "/home/worker/app/scout/server/app.py", line 113, in configure_extensions
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:     extensions.chanjo_report.init_app(app)
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:   File "/home/worker/app/scout/server/extensions/chanjo_extension.py", line 32, in init_app
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:     raise ImportError(
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]: ImportError: An SQL db path was given, but chanjo-report could not be registered.
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]: [2024-11-28 15:41:12 +0100] [10] [INFO] Worker exiting (pid: 10)
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]: [2024-11-28 15:41:12 +0100] [11] [ERROR] Exception in worker process
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]: Traceback (most recent call last):
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:   File "/venv/lib/python3.12/site-packages/gunicorn/arbiter.py", line 608, in spawn_worker
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:     worker.init_process()
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:   File "/venv/lib/python3.12/site-packages/gunicorn/workers/gthread.py", line 94, in init_process
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:     super().init_process()
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:   File "/venv/lib/python3.12/site-packages/gunicorn/workers/base.py", line 135, in init_process
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:     self.load_wsgi()
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:   File "/venv/lib/python3.12/site-packages/gunicorn/workers/base.py", line 147, in load_wsgi
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:     self.wsgi = self.app.wsgi()
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:                 ^^^^^^^^^^^^^^^
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:   File "/venv/lib/python3.12/site-packages/gunicorn/app/base.py", line 66, in wsgi
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:     self.callable = self.load()
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:                     ^^^^^^^^^^^
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:   File "/venv/lib/python3.12/site-packages/gunicorn/app/wsgiapp.py", line 57, in load
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:     return self.load_wsgiapp()
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:            ^^^^^^^^^^^^^^^^^^^
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:   File "/venv/lib/python3.12/site-packages/gunicorn/app/wsgiapp.py", line 47, in load_wsgiapp
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:     return util.import_app(self.app_uri)
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:   File "/venv/lib/python3.12/site-packages/gunicorn/util.py", line 370, in import_app
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:     mod = importlib.import_module(module)
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:   File "/usr/local/lib/python3.12/importlib/__init__.py", line 90, in import_module
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:     return _bootstrap._gcd_import(name[level:], package, level)
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:   File "<frozen importlib._bootstrap>", line 1387, in _gcd_import
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:   File "<frozen importlib._bootstrap>", line 1360, in _find_and_load
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:   File "<frozen importlib._bootstrap>", line 1331, in _find_and_load_unlocked
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:   File "<frozen importlib._bootstrap>", line 935, in _load_unlocked
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:   File "<frozen importlib._bootstrap_external>", line 995, in exec_module
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:   File "<frozen importlib._bootstrap>", line 488, in _call_with_frames_removed
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:   File "/home/worker/app/scout/server/auto.py", line 13, in <module>
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:     app = create_app(config_file=os.environ["SCOUT_CONFIG"])
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:   File "/home/worker/app/scout/server/app.py", line 68, in create_app
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:     configure_extensions(app)
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:   File "/home/worker/app/scout/server/app.py", line 113, in configure_extensions
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:     extensions.chanjo_report.init_app(app)
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:   File "/home/worker/app/scout/server/extensions/chanjo_extension.py", line 32, in init_app
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]:     raise ImportError(
Nov 28 15:41:12 cg-services-stage.scilifelab.se podman[3101366]: ImportError: An SQL db path was given, but chanjo-report could not be registered.
```
Now for `chanjo-report` `setuptools`.

**Comment by dnil** [comment]
3.11 builds have 
```
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]:     from . import extensions
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]:   File "/home/worker/app/scout/server/extensions/__init__.py", line 3, in <module>
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]:     from authlib.integrations.flask_client import OAuth
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]:   File "/venv/lib/python3.11/site-packages/authlib/integrations/flask_client/__init__.py", line 2, in <module>
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]:     from .integration import FlaskIntegration, token_update
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]:   File "/venv/lib/python3.11/site-packages/authlib/integrations/flask_client/integration.py", line 3, in <module>
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]:     from ..base_client import FrameworkIntegration
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]:   File "/venv/lib/python3.11/site-packages/authlib/integrations/base_client/__init__.py", line 3, in <module>
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]:     from .sync_openid import OpenIDMixin
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]:   File "/venv/lib/python3.11/site-packages/authlib/integrations/base_client/sync_openid.py", line 1, in <module>
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]:     from authlib.jose import jwt, JsonWebToken, JsonWebKey
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]:   File "/venv/lib/python3.11/site-packages/authlib/jose/__init__.py", line 14, in <module>
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]:     from .rfc7517 import Key, KeySet, JsonWebKey
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]:   File "/venv/lib/python3.11/site-packages/authlib/jose/rfc7517/__init__.py", line 10, in <module>
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]:     from ._cryptography_key import load_pem_key
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]:   File "/venv/lib/python3.11/site-packages/authlib/jose/rfc7517/_cryptography_key.py", line 1, in <module>
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]:     from cryptography.x509 import load_pem_x509_certificate
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]:   File "/venv/lib/python3.11/site-packages/cryptography/x509/__init__.py", line 7, in <module>
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]:     from cryptography.x509 import certificate_transparency, verification
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]:   File "/venv/lib/python3.11/site-packages/cryptography/x509/certificate_transparency.py", line 8, in <module>
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]:     from cryptography.hazmat.bindings._rust import x509 as rust_x509
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]: ImportError: /lib/x86_64-linux-gnu/libc.so.6: version `GLIBC_2.33' not found (required by /venv/lib/python3.11/site-packages/cryptography/hazmat/bindings/_rust.abi3.so)
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]: [2024-11-28 15:29:30 +0100] [12] [INFO] Worker exiting (pid: 12)
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]: [2024-11-28 15:29:30 +0100] [8] [ERROR] Worker (pid:11) exited with code 3
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]: Traceback (most recent call last):
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]:   File "/venv/lib/python3.11/site-packages/gunicorn/arbiter.py", line 208, in run
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]:     self.sleep()
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]:   File "/venv/lib/python3.11/site-packages/gunicorn/arbiter.py", line 359, in sleep
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]:     ready = select.select([self.PIPE[0]], [], [], 1.0)
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]:             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]:   File "/venv/lib/python3.11/site-packages/gunicorn/arbiter.py", line 241, in handle_chld
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]:     self.reap_workers()
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]:   File "/venv/lib/python3.11/site-packages/gunicorn/arbiter.py", line 529, in reap_workers
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]:     raise HaltServer(reason, self.WORKER_BOOT_ERROR)
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]: gunicorn.errors.HaltServer: <HaltServer 'Worker failed to boot.' 3>
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]: During handling of the above exception, another exception occurred:
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]: Traceback (most recent call last):
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]:   File "/venv/bin/gunicorn", line 8, in <module>
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]:     sys.exit(run())
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]:              ^^^^^
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]:   File "/venv/lib/python3.11/site-packages/gunicorn/app/wsgiapp.py", line 66, in run
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]:     WSGIApplication("%(prog)s [OPTIONS] [APP_MODULE]", prog=prog).run()
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]:   File "/venv/lib/python3.11/site-packages/gunicorn/app/base.py", line 235, in run
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]:     super().run()
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]:   File "/venv/lib/python3.11/site-packages/gunicorn/app/base.py", line 71, in run
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]:     Arbiter(self).run()
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]:   File "/venv/lib/python3.11/site-packages/gunicorn/arbiter.py", line 228, in run
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]:     self.halt(reason=inst.reason, exit_status=inst.exit_status)
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]:   File "/venv/lib/python3.11/site-packages/gunicorn/arbiter.py", line 341, in halt
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]:     self.stop()
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]:   File "/venv/lib/python3.11/site-packages/gunicorn/arbiter.py", line 395, in stop
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]:     time.sleep(0.1)
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]:   File "/venv/lib/python3.11/site-packages/gunicorn/arbiter.py", line 241, in handle_chld
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]:     self.reap_workers()
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]:   File "/venv/lib/python3.11/site-packages/gunicorn/arbiter.py", line 529, in reap_workers
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]:     raise HaltServer(reason, self.WORKER_BOOT_ERROR)
Nov 28 15:29:30 cg-services-stage.scilifelab.se podman[3098470]: gunicorn.errors.HaltServer: <HaltServer 'Worker failed to boot.' 3>
Nov 28 15:29:31 cg-services-stage.scilifelab.se systemd[1524]: scout@panelapp_links.service: Main process exited, code=exited, status=1/FAILURE
Nov 28 15:29:31 cg-services-stage.scilifelab.se podman[3098567]: scout-panelapp_links
Nov 28 15:29:31 cg-services-stage.scilifelab.se systemd[1524]: scout@panelapp_links.service: Failed with result 'exit-code'.
```

File: scout/server/extensions/panelapp_extension.py [hunk 1]
```diff
@@ -41,7 +41,7 @@ def set_panel_types(self, json_panels: dict):
             for type in panel.get("types", []):
                 self.panel_types.add(type["slug"])
 
-    def get_panel_ids(self, signed_off: bool) -> list[int]:
+    def get_panel_ids(self, signed_off: bool) -> List[int]:
```
Comment (dnil) [inline scout/server/extensions/panelapp_extension.py:44]: 👍

**Review by dnil — APPROVED** [review]
Lets go, given that we update to 312 once we have chanjo etc.

**Pull request merged**